source/Octopus.Client/Repositories/UserRepository.cs

```diff
@@ -88,10 +89,10 @@ public UserResource GetCurrent()
             return Client.Get<UserResource>(Client.RootDocument.Link("CurrentUser"));
         }
 
-        public UserPermissionSetResource GetPermissions(UserResource user)
+        public UserPermissionSetResource GetPermissions(UserResource user, string spaceId)
```

Comment:
Does this need a default of null like all the other actions?

Reply:
good catch